tests/unit_tests/tasks/test_cron_util.py

```diff
@@ -53,34 +56,40 @@ def test_cron_schedule_window_los_angeles(
     Reports scheduler: Test cron schedule window for "America/Los_Angeles"
     """
 
-    with freeze_time(current_dttm):
-        datetimes = cron_schedule_window(cron, "America/Los_Angeles")
-        assert (
-            list(cron.strftime("%A, %d %B %Y, %H:%M:%S") for cron in datetimes)
-            == expected
-        )
+    datetimes = cron_schedule_window(
+        datetime.fromisoformat(current_dttm), cron, "America/Los_Angeles"
+    )
+    assert (
+        list(cron.strftime("%A, %d %B %Y, %H:%M:%S") for cron in datetimes) == expected
+    )
 
 
 @pytest.mark.parametrize(
     "current_dttm, cron, expected",
     [
-        ("2020-01-01T00:59:01Z", "0 1 * * *", []),
```

Comment:
Why are we removing the tests which use the zone designator for the zero UTC offset?

Reply:
They're not removed, I just changed the timestamp format to ISO (zero UTC offset represented by `+00:00`).  The format used is inconsequential to the `cron_schedule_window` function being tested, since it's just passed a datetime object.